parm/config/gfs/config.resources

```diff
@@ -791,313 +760,312 @@ elif [[ "${step}" = "metp" ]]; then
-    npe_node_ecen=$(echo "${npe_node_max} / ${nth_ecen}" | bc)
-    export npe_node_ecen
+    if [[ "${machine}" == "HERA" ]]; then export nth_ecen=6; fi
+    if [[ ${CASE} == "C384" || ${CASE} == "C192" || ${CASE} == "C96" || ${CASE} == "C48" ]]; then
+      export nth_ecen=2
+    fi
+    export npe_node_ecen=$(( npe_node_max / nth_ecen ))
     export nth_cycle=${nth_ecen}
-    npe_node_cycle=$(echo "${npe_node_max} / ${nth_cycle}" | bc)
-    export npe_node_cycle
+    export npe_node_cycle=$(( npe_node_max / nth_cycle ))
     export is_exclusive=True
+    ;;
 
-elif [[ ${step} = "esfc" ]]; then
-
+  "esfc")
     export wtime_esfc="00:08:00"
     export npe_esfc=80
     export nth_esfc=1
-    npe_node_esfc=$(echo "${npe_node_max} / ${nth_esfc}" | bc)
-    export npe_node_esfc
+    export npe_node_esfc=$(( npe_node_max / nth_esfc ))
     export nth_cycle=${nth_esfc}
-    npe_node_cycle=$(echo "${npe_node_max} / ${nth_cycle}" | bc)
-    export npe_node_cycle
-    export memory_esfc="80GB"
-
-elif [[ ${step} = "epos" ]]; then
+    export npe_node_cycle=$(( npe_node_max / nth_cycle ))
+    export memory_esfc="8GB"
+    ;;
 
+  "epos")
     export wtime_epos="00:15:00"
     export npe_epos=80
     export nth_epos=1
-    npe_node_epos=$(echo "${npe_node_max} / ${nth_epos}" | bc)
-    export npe_node_epos
+    export npe_node_epos=$(( npe_node_max / nth_epos ))
     export is_exclusive=True
+    ;;
 
-elif [[ ${step} = "postsnd" ]]; then
-
+  "postsnd")
     export wtime_postsnd="02:00:00"
     export npe_postsnd=40
     export nth_postsnd=8
     export npe_node_postsnd=10
     export npe_postsndcfp=9
     export npe_node_postsndcfp=1
-    postsnd_req_cores=$(echo "${npe_node_postsnd} * ${nth_postsnd}" | bc)
-    if [[ ${postsnd_req_cores} -gt "${npe_node_max}" ]]; then
-        npe_node_postsnd=$(echo "${npe_node_max} / ${nth_postsnd}" | bc)
-        export npe_node_postsnd
+    postsnd_req_cores=$(( npe_node_postsnd * nth_postsnd ))
+    if (( postsnd_req_cores > npe_node_max )); then
+        export npe_node_postsnd=$(( npe_node_max / nth_postsnd ))
     fi
     export is_exclusive=True
+    ;;
 
-elif [[ ${step} = "awips" ]]; then
-
+  "awips")
     export wtime_awips="03:30:00"
     export npe_awips=1
     export npe_node_awips=1
     export nth_awips=1
-    export memory_awips="3GB"
-
-elif [[ ${step} = "npoess" ]]; then
+    export memory_awip="3GB"
+    ;;
 
+  "npoess")
     export wtime_npoess="03:30:00"
     export npe_npoess=1
     export npe_node_npoess=1
     export nth_npoess=1
-    export memory_npoess="3GB"
-
-elif [[ ${step} = "gempak" ]]; then
+    export memory_npoes="3GB"
+    ;;
 
+  "gempak")
     export wtime_gempak="03:00:00"
     export npe_gempak=2
     export npe_gempak_gfs=28
     export npe_node_gempak=2
     export npe_node_gempak_gfs=28
     export nth_gempak=1
     export memory_gempak="4GB"
-    export memory_gempak_gfs="2GB"
-
-elif [[ ${step} = "mos_stn_prep" ]]; then
+    export memory_gempak_gfs="GB"
```

Comment:
```suggestion
    export memory_gempak_gfs="2GB"
```

Reply:
Fixed, and I went through the remaining memory lines.